### PR TITLE
Core/ts version task 176 support contract v3

### PR DESCRIPTION
**Pull request by hypersolo**
## What does this MR do?

handle new version of contract

## Check list  

- [ ] All Merge Request pipelines should be passed.
- [ ] Make sure all the changes are necessary and correct.
- [ ] Make sure no needed change missing (all necessary changes are made).
- [ ] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [ ] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: src/test/e2e/orderbook-e2e.test.ts [hunk 1]
```diff
@@ -27,6 +27,11 @@ describe("Get /orderbook endpoint", () => {
     spiedSetInterval.mockImplementation(jest.fn());
     providers = Providers.getInstance();
     await providers.startProviders();
+    providers.getProviders().forEach((provider) => {
```
Comment (WilliamHawkingStephen) [inline src/test/e2e/orderbook-e2e.test.ts:30]: - maybe we can use beforeEach for remove these duplicates :heart:

File: src/data/database/postgres/migration/019-drop-max-fee-ratio-column-from-networks-table.migration.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports = {
   async down(queryInterface: QueryInterface) {
     await queryInterface.addColumn(tableName, "max_fee_ratio", {
       defaultValue: 30,
-      allowNull: false,
+      allowNull: true,
```
Comment (WilliamHawkingStephen) [inline src/data/database/postgres/migration/019-drop-max-fee-ratio-column-from-networks-table.migration.ts:12]: - why we set allowNull for max_fee_ration true ?

Comment (hypersolo) [reply src/data/database/postgres/migration/019-drop-max-fee-ratio-column-from-networks-table.migration.ts:12]: zksync :rose:

File: src/test/e2e/orderbook-e2e.test.ts (hunk 1 above)
Comment (hypersolo) [reply src/test/e2e/orderbook-e2e.test.ts:30]: i want to mock this function after providers are initiated and started.
we can;t move it there :tulip:

**Review by WilliamHawkingStephen — APPROVED** [review]

**Review by franken5tein — APPROVED** [review]

**Pull request merged**